laravel-svelte-port/svelte-ui/src/routes/app/accounts/[accountId]/settings/assignment-policy/+page.svelte

```
@@ -0,0 +1,85 @@
+  let enforceAgentCapacity = $state(false);
+  let useOnlineAgentsOnly = $state(true);
+  let isSaving = $state(false);
+
+  $effect(() => {
+    const settings = authStore.uiSettings?.assignmentPolicy || {};
+    roundRobinEnabled = settings.roundRobinEnabled ?? true;
+    enforceAgentCapacity = settings.enforceAgentCapacity ?? false;
+    useOnlineAgentsOnly = settings.useOnlineAgentsOnly ?? true;
+  });
+
+  async function handleSave() {
```

ESLint

[error] 'handleSave' is defined but never used.

(no-unused-vars)